dbt_meshify/main.py

```diff
@@ -65,40 +68,36 @@ def connect(projects_dir):
 
 
 @cli.command(name="split")
+@create_path
+@click.argument("project_name")
 @exclude
 @project_path
 @select
 @selector
-def split():
+def split(project_name, select, exclude, project_path, selector, create_path):
```

Comment:
Since `select` now allows multiple arguments, we cannot have `--select` before the `project_name` argument.

```console
(dbt-meshify-py3.11) > $ poetry run dbt-meshify split --select "+orders" revenue                                                                                                                                         
Usage: dbt-meshify split [OPTIONS] PROJECT_NAME
Try 'dbt-meshify split --help' for help.

Error: Missing argument 'PROJECT_NAME'.
```

Instead, we need to order arguments/options specifically

```
(dbt-meshify-py3.11) > $ poetry run dbt-meshify split revenue --select "+orders"                                                                                                                                   
```

I don't this is a blocker per se. At the very least, documentation should be refined in a follow-up.